### DISCUSSION TITLE
I can't build ChocolateyGUI wtih a NuGet exception

### DISCUSSION THREAD

**Discussion by buzhi-kefou** [cake-build/cake#4162] [Q&A]
Who can help me fix this problem? I have cloned the git repository with command.
<img width="960" alt="6B7C637C-D09C-489f-933B-970CB4F73E74" src="https://github.com/cake-build/cake/assets/61810948/d90eba66-4c35-4ed4-b53b-3e1c922c5cd1">

**Comment by gep13** [comment]
@buzhi-kefou for future questions like this, it is best to reach out directly to the project that you are trying to build.  Yes, Cake is being used to build Chocolatey GUI, but the Cake team is not responsible for any problems that you might be having with that build.

For the specific problem that you are facing though, I can provide some assistance.  Chocolatey GUI takes a dependency on the NuGet Package for chocolatey.lib, specifically, version 2.0.0.  This package version hasn't actually been released yet, therefore it is not available on nuget.org yet.  That is why your build is failing.  Once the Chocolatey CLI 2.0.0 release is complete, you should be able to run the build successfully.

**Reply by buzhi-kefou** [reply]
sorry, I am the beginner of C#,how can i clear the build cache? i changed the package's version of chocolatey.lib, but the error still here.
